hackernews-react-apollo/src/gql/addUser.js

```diff
@@ -0,0 +1,10 @@
+import { gql } from "@apollo/client";
+
+export const ADD_USER = gql`
```

Comment:
you dont need a different file for each query. You can have userQuery, appQuery etc and aggregate similar ones into same file